packages/core/content-manager/admin/src/components/AttributeFilter/utils/getAttributeType.js

```diff
@@ -0,0 +1,13 @@
+import get from 'lodash/get';
+
+const getAttributeType = (attributeName, contentType, metaData) => {
```

Comment:
The string input does not exist I prefer to return a text instead WDYT?

Reply:
I did not understand the comment, sorry 😅